### ISSUE TITLE
Couldn't run through npx

### ISSUE THREAD

**Issue by romash1408** [amalfra/junit-cli-report-viewer#9]
```bash
$ node -v
v14.18.0
$ npx junit-cli-report-viewer report.xml
npx: installed 20 in 5.734s
fs.js:497
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, open 'package.json'
    at Object.openSync (fs.js:497:3)
    at Object.readFileSync (fs.js:393:35)
    at file:///home/username/.npm/_npx/24182/lib/node_modules/junit-cli-report-viewer/index.js:9:35
    at ModuleJob.run (internal/modules/esm/module_job.js:183:25)
    at async Loader.import (internal/modules/esm/loader.js:178:24)
    at async Object.loadESM (internal/process/esm_loader.js:68:5) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'package.json'
}
```

**Comment by amalfra** [comment]
@romash1408 Thanks for reporting. This looks like a bug after migrating to ES6 imports. I am working on the fix now.

**Comment by amalfra** [comment]
@romash1408 This has been fixed and a new release [v1.5.1](https://github.com/amalfra/junit-cli-report-viewer/releases/tag/1.5.1) has been done. Please install that and it should start working properly.
